core/src/apps/stellar/operations/layout.py

```diff
@@ -110,6 +123,19 @@
     await confirm_asset_issuer(ctx, op.buying_asset)
 
 
+async def _confirm_buy_offer(ctx, title, op):
```

Comment:
please extend and reuse `_confirm_offer` instead

Reply:
Looking into it, `_confirm_offer` and `_confirm_buy_offer` have few lines in common (4 out of 8). Whatever way you put it, the relation between buying asset & selling asset is inversed - so is the price - so I see no way to rewrite `_confirm_offer` without crippling code readability.

Reply:
point taken, let's keep it